Automattic/wp-calypso
# Checkout: add NetBanking payments support via dLocal

#### Changes proposed in this Pull Request

* Adding support for new payment method (Netbanking via dLocal).
* Refactored ebanx validation rules so that they can be reused for other countries as well

#### Testing instructions

* Requires backend D23563-code with store-sandbox on.
* Use `AAAPL1234C` as your PAN number (or test validation failure for of any other format).
* You should be redirected to a test page when you can confirm or decline the payment. Note that you won't be redirected back and that the payment won't be completed on wpcom during testing.
 
#### Screenshots
<img width="692" alt="Screen Shot 2019-04-17 at 16 07 51" src="https://user-images.githubusercontent.com/844866/56290061-02349800-612b-11e9-997f-ce0127870a3e.png">
